src/engraving/style/styledef.cpp

```diff
@@ -499,7 +499,7 @@ const std::array<StyleDef::StyleValue, size_t(Sid::STYLES)> StyleDef::styleValue
     styleDef(chordModifierMag,                           PropertyValue(1.0)),
     styleDef(chordModifierAdjust,                        PropertyValue(0.0)),
     styleDef(verticallyStackModifiers,                   false),
-    styleDef(chordStackedModiferMag,                     PropertyValue(0.75)),
+    styleDef(chordStackedModifierMag,                    PropertyValue(0.75)),
```

Comment:
This is a breaking change for compatibility, but since Sid is new in 4.6, we can go ahead with it, _if we're quick enough for beta_